digdag-tests/src/test/java/acceptance/ServerScheduleIT.java

```diff
@@ -456,10 +454,10 @@ public void testSkipAndEnable()
             client.disableSchedule(daily.getId());
             client.enableSchedule(daily.getId(), true, skipToTime);
             // get schedule
-            RestSchedule enabled = client.getSchedule(daily.getId());
-            assertThat(enabled.getDisabledAt(), is(Optional.absent()));
-            assertThat(enabled.getNextRunTime(), is(Instant.parse("2291-02-09T10:00:00Z")));
-            assertThat(enabled.getNextScheduleTime(), is(OffsetDateTime.parse("2291-02-09T00:00Z")));
+            RestSchedule enabled1 = client.getSchedule(daily.getId());
```

Comment:
Why do you rename this? `enabled` seems to be already isolated in this local scope without conflict.

Reply:
I have no specific reason, yes so I will rename this again.

Reply:
Updated: 8422e41